src/formDemoMockResponse.js

```diff
@@ -121,7 +121,7 @@ const metadata = {
       isAggregatable: false,
       visible: true,
       description:
-        'Complex description with http://molgenis.org url\nAnd more text here www.molgenis.org Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.{i}test http://molgenis.org more test{/i}'
+        'Complex description with http://molgenis.org url\nAnd more text here www.molgenis.org {br}• list item 1{br}• list item 2{br}• list item 3{br}Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.{i}test http://molgenis.org more test{/i}'
```

Comment:
There comes a point that we want separate test fields for each use case. I'm fine with not doing that yet (molgenis eol and all).